hostscripts/soc-ci/soc-ci

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+# Copyright (c) 2016 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+
+import getpass
+import os
+import re
+import sys
+
+from six.moves import configparser
+from six.moves import input
+
+from jenkinsapi.jenkins import Jenkins
```

Comment:
flake8 says
hostscripts/soc-ci/soc-ci:29:1: I100 Import statements are in the wrong order. from jenkinsapi.jenkins should be before from six.moves